lib/huginn_acumen_product_agent/acumen_product_agent.rb

```diff
@@ -104,7 +109,19 @@ def handle(event)
             end
 
             products.each do |product|
-                create_event payload: product
+                if (ignore_skus.empty?)
+                  create_event payload: product
+                else
+                  emit_product = true
+                  ignore_skus.each do |ignore_sku|
+                    if (ignore_sku == product['sku'])
+                      emit_product = false
+                    end
+                  end
+                  if (emit_product)
+                    create_event payload: product
+                  end
+                end
```

Comment:
```
unless ignore_skus.include?(product['sku'])
    create_event payload: product
end
```

Reply:
@mrogers-5s This should be fixed let me know what you think.